autoload/beancount.vim

```diff
@@ -56,14 +66,29 @@ function! beancount#complete_account(findstart, base)
         endif
         let b:beancount_accounts = beancount#find_accounts(l:root)
     endif
-    let l:pattern = '\V' . substitute(a:base, ":", '\\S\\*:\\S\\*', "g")
+
+    if g:beancount_account_completion ==? 'chunks'
+        let l:pattern = '\V' . substitute(a:base, ":", '\\[^ \\t:]\\*:', "g") . '\[^ \t:]\*'
```

Comment:
I don't mind having them together, I just wanted to make sure 1) it was intentional and 2) I understood what you were doing.